<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawContour</class>
 <widget class="QWidget" name="DrawContour">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1836</width>
    <height>1006</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DrawContour</string>
  </property>
  <widget class="QPushButton" name="pushButton_open">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="frame_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>1781</width>
     <height>901</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
